buildbot-host/buildmaster/worker-default.ini

```diff
@@ -111,7 +111,7 @@ image=openvpn_community/buildbot-worker-fedora-rawhide:v1.1.0
 enable_debian_builds=false
 
 [opensuse-leap-15]
-image=openvpn_community/buildbot-worker-opensuse-leap-15:v1.0.2
+image=openvpn_community/buildbot-worker-opensuse-leap-15:v1.0.3
```

Comment:
Not actually relevant for this PR. But we forgot to update it and that actually hid the error because the new opensuse containers with mbedTLS v3 weren't used. So this problem started this whole investigation. If you prefer I can put the change in a separate commit with explanation.